PacketDefinitions420/PacketHandlerManager.cs

```diff
@@ -244,8 +244,16 @@ public bool HandlePacket(Peer peer, byte[] data, Channel channelId)
         public bool HandleDisconnect(Peer peer)
         {
             ulong playerId = _peers.FirstOrDefault(x => x.Value.Address.Equals(peer.Address)).Key;
-            // TODO: fix all to use ulong
-            return _game.HandleDisconnect(playerId);
+            var player = _game.PlayerManager.GetPlayers().Find(x => x.Item2.PlayerId == playerId).Item2;
+            var peerInfo = _game.PlayerManager.GetPeerInfo(player.PlayerId);
+
+            if (peerInfo != null)
+            {
+                _game.PacketNotifier.NotifyUnitAnnounceEvent(UnitAnnounces.SUMMONER_DISCONNECTED, peerInfo.Champion);
```

Comment:
Future suggestion: Maybe we want to take this line inside champion at some point. Currently, it is not possible as `peerInfo==null` is possible with disconnect after disconnect maybe???